autopush/tests/test_router.py

```diff
@@ -296,9 +296,177 @@ def check_results(result):
         return d
 
 
+class GCMClientTestCase(unittest.TestCase):
```

Comment:
Could we put this TestCase in a new file? test_router's already pretty hefty

Reply:
👍